airbyte-integrations/connectors/source-facebook-marketing/integration_tests/spec.json

```diff
@@ -30,13 +30,6 @@
         "title": "Authentication",
         "description": "Credentials for connecting to the Facebook Marketing API",
         "type": "object",
-        "discriminator": {
```

Comment:
what is the discriminator used for?

Reply:
It is used to distinguish between different schema definitions based on a property value. In this case I think this is a little overloaded as we also are using also OneOf to have one of either schemas